src/st/dre/161DC.c

```diff
@@ -1,5 +1,7 @@
 #include "dre.h"
 
+extern s32 D_80097400[];
```

Comment:
I just had a look at `build/us/stdre.map` and `80097400` is right below `g_unkGraphicsStruct`. Infact that results to be the field `unk0` from that struct!
```suggestion
```

Reply:
Thanks, fixed!